src/runtime_src/core/tools/common/reports/ReportAie.cpp

```diff
@@ -87,11 +87,13 @@ writeReport(const xrt_core::device* /*_pDevice*/,
     }
   }
 
+  _output << "Aie\n";
   // validate and print aie metadata by checking schema_version node
-  if (!_pt.get_child_optional("aie_metadata.schema_version"))
+  if (!_pt.get_child_optional("aie_metadata.schema_version")) {
+    _output << "  <AIE information unavailable>" << std::endl << std::endl;
```

Comment:
Use the same string in both console and in json output. "AIE information is not available"